src/Solver/Interoperability.cpp

```diff
@@ -537,24 +537,23 @@ void seissol::Interoperability::setMaterial(int i_meshId, int i_side, double* i_
 
 #ifdef USE_PLASTICITY
 void seissol::Interoperability::setInitialLoading( int i_meshId, double *i_initialLoading ) {
-  PlasticityData& plasticity = m_ltsLut.lookup(m_lts->plasticity, i_meshId - 1);
+  PlasticityData* plasticity = &m_ltsLut.lookup(m_lts->plasticity, i_meshId - 1);
```

Comment:
Why do you change change a reference to a pointer here?

Reply:
If I do not change to a pointer, the plasticity data in the tree is never actually written. I thought this would be the easiest way. Compare with seissol::Interoperability::setMaterial a few lines above.

Reply:
No, this does not make sense. A reference is nothing else than a pointer that you cannot change. That is also the reason why a pointer is used in setMaterial.

Reply:
There's also another advantage: References are not null (unless you **really** want them to be).